config/sync/media.type.embeddable.yml

```diff
@@ -0,0 +1,35 @@
+uuid: 04580fd6-ab11-4556-ab0d-c3d8f1e6acbc
+langcode: en
+status: true
+dependencies:
+  module:
+    - stanford_media
+id: embeddable
+label: Embeddable
+description: 'Stanford Embeds'
+source: embeddable
+queue_thumbnail_downloads: false
+new_revision: false
+source_configuration:
+  thumbnails_directory: 'public://oembed_thumbnails'
+  providers:
```

Comment:
Why don't these pull from the eoembed providers module config.?

Reply:
So, for oEmbeds, you have to hard-code a list of providers that the plugin may support (https://github.com/SU-SWS/stanford_media/pull/61/files/53523547773564b25447fdb24c0f6a168f3f8bc7#r483181964) , and then in the config for the media type, you have the option enable or disable each of those supported providers individually.  These values determine which of our providers we will turn on.